packages/stg-evm-v2/src/peripheral/oft-wrapper/OFTWrapper.sol

```diff
@@ -230,6 +233,64 @@ contract OFTWrapper is IOFTWrapper, Ownable, ReentrancyGuard {
         );
     }
 
+    function epv2_sendOFT(
+        address _oft,
+        epv2_SendParam calldata _sendParam,
+        epv2_MessagingFee calldata _fee,
+        address _refundAddress,
+        FeeObj calldata _feeObj
+    ) external payable nonReentrant {
+        /// @dev de-dust and transfer to the wrapper as an intermediate step.
+        uint256 decimalConversionRate = epv2_OFT(_oft).decimalConversionRate();
+        uint256 amountToSwap = (_getAmountAndPayFee(_oft, _sendParam.amountLD, _sendParam.minAmountLD, _feeObj) /
+            decimalConversionRate) * decimalConversionRate;
```

Comment:
Here I'd maybe do the multiplication before the division not to lose precision

Reply:
It was done to de-dust.